### ISSUE TITLE
The color of comment incorrectly when the scope is under meta.class

### ISSUE THREAD

**Issue by ooJerryLeeoo** [braver/Solarized#62]
Here's an example:

![screen shot 2017-07-19 at 19 18 39](https://user-images.githubusercontent.com/20783502/28364534-6602f8f0-6cb7-11e7-85f8-92b695e4211c.png)

the scope of `//` is:
```
source.c++ meta.class.c++ meta.block.c++ comment.line.double-slash.c punctuation.definition.comment.c
```

and I think the problem is come from `JavaScript punctation inside class`.

**Comment by braver** [comment]
Good catch, that specific block was super sloppy. Don't know how that slipped through the cracks really. Should be fixed in 1.5.10

**Comment by ooJerryLeeoo** [comment]
Hi @braver, I was wondering why change the `meta.class` color to yellow instead of the text color ?
